extensions/amp-story-education/0.1/amp-story-education.js

```diff
@@ -217,13 +249,46 @@ export class AmpStoryEducation extends AMP.BaseElement {
     }
 
     this.storeService_.dispatch(Action.TOGGLE_PAUSED, true);
+    this.storeService_.dispatch(Action.TOGGLE_EDUCATION, true);
 
     this.mutateElement(() => {
       removeChildren(this.containerEl_);
       toggle(this.containerEl_, true);
       this.containerEl_.appendChild(template);
     });
   }
+
+  /**
+   * Asks the viewer whether the navigation screen should be shown.
+   * @param {!Screen} screen Screen to show
+   * @param {!State} state State to set
+   * @private
+   */
+  maybeShowScreen_(screen, state) {
+    // Only show education screens when the ampdoc is visible.
+    this.getAmpDoc()
+      .whenFirstVisible()
+      .then(() => {
+        // TODO(gmajoulet): update this method to support showing multiple
+        // screens, if/when needed.
+        this.viewer_
+          .sendMessageAwaitResponse(
+            'canShowScreens',
```

Comment:
nit: I'm not entirely sure about making this plural; this seems like it could lead us to want to request multiple and cache the result, whereas I think the idea is to lookup can show at the instant the screen is to be displayed, in case things have changed

The inner object notation LGTM, but we should document

Reply:
The motivation behind making it plural is that we want to know how many total screens we are going to display at a given time to:
1. Have accurate "Tip 1/n" labels
2. Not animate out / animate in again between different screens

Reply:
But, this is one "screen" right?  So (1) can't the tip labels be hardcoded? (2) would it do that?

Reply:
Maybe I'm over engineering this: if tomorrow we add more screens, say audio controls, we might ask players if they want to show the navigation screens, and the audio controls screen on load.
If we don't ask that in the same message, there might be a delay in between both screens causing (2), and inaccurate "Tip 1/n" hardcoded labels

Reply:
I guess I'm also fine with an array and a comment on the format side that says we shouldn't cache these values

Reply:
Sounds good.
The format side should always display all the screens right away (one by one), but not store the response to be used later.